imports/plugins/core/orders/client/components/orderTable.js

```diff
@@ -97,7 +98,8 @@ class OrderTable extends Component {
 
   renderOrderInfo(order) {
     const { displayMedia, moment } = this.props;
-    const { invoice } = getPaymentForCurrentShop(order);
+    const [payment] = order.payments || [];
+    const { amount } = payment || {};
```

Comment:
Can you explain how this is working to me? I think it's just a syntax I haven't seen before, but if this is a payment in an array, how are we extracting amount directly from payment?

Reply:
`const [payment]` takes the first payment and saves it in the `payment` const (array destructuring). Then I destructure the `amount` property off `payment`.

Ideally we shouldn't be using only the first payment anywhere, but I was trying to avoid doing too many changes in this one PR.